src/Evolution/Systems/GrMhd/ValenciaDivClean/Subcell/NeighborPackagedData.hpp

```diff
@@ -0,0 +1,231 @@
+                evolution::dg::Actions::detail::NormalVector<3>>>>;
+
+        const auto& element = db::get<domain::Tags::Element<3>>(box);
+        const auto& eos = get<hydro::Tags::EquationOfStateBase>(box);
+
+        for (const auto& mortar_id : mortars_to_reconstruct_to) {
+          const Direction<3>& direction = mortar_id.first;
+
+          Index<3> extents = subcell_mesh.extents();
+          ++extents[direction.dimension()];
+
+          // Computed prims and cons on face via reconstruction
+          const size_t num_face_pts = subcell_mesh.extents()
+                                          .slice_away(direction.dimension())
+                                          .product();
+          Variables<dg_package_data_argument_tags> vars_on_face{num_face_pts};
+          // Copy spacetime vars over from volume.
+          using spacetime_vars_to_copy = tmpl::list<
+              gr::Tags::Lapse<DataVector>,
+              gr::Tags::Shift<3, Frame::Inertial, DataVector>,
+              gr::Tags::SpatialMetric<3>,
+              gr::Tags::SqrtDetSpatialMetric<DataVector>,
+              gr::Tags::InverseSpatialMetric<3, Frame::Inertial, DataVector>>;
+          tmpl::for_each<spacetime_vars_to_copy>(
+              [&direction, &extents, &vars_on_face,
+               &spacetime_vars_on_faces =
+                   db::get<evolution::dg::subcell::Tags::OnSubcellFaces<
+                       typename System::flux_spacetime_variables_tag, 3>>(box)](
+                  auto tag_v) noexcept {
+                using tag = tmpl::type_from<decltype(tag_v)>;
+                data_on_slice(make_not_null(&get<tag>(vars_on_face)),
+                              get<tag>(gsl::at(spacetime_vars_on_faces,
+                                               direction.dimension())),
+                              extents, direction.dimension(),
+                              direction.side() == Side::Lower
+                                  ? 0
+                                  : extents[direction.dimension()] - 1);
+              });
+
+          call_with_dynamic_type<void, typename grmhd::ValenciaDivClean::fd::
+                                           Reconstructor::creatable_classes>(
+              &recons, [&element, &eos, &mortar_id, &nhbr_subcell_data,
+                        &subcell_mesh, &vars_on_face,
+                        &volume_prims](const auto& reconstructor) noexcept {
+                reconstructor->reconstruct_fd_neighbor(
+                    make_not_null(&vars_on_face), volume_prims, eos, element,
+                    nhbr_subcell_data, subcell_mesh, mortar_id.first);
+              });
+
+          grmhd::ValenciaDivClean::subcell::compute_fluxes(
+              make_not_null(&vars_on_face));
+
+          // Note: since the spacetime isn't dynamical we don't need to
+          // worry about different normal vectors on the different sides
+          // of the element.
```

Comment:
In the case of a dynamical spacetime, would the solution be to send the `NormalCovector` in the packaged data as well? Could you add another sentence explaining that?